aemiers/rancid-tomatillos
# Feature/click details

## Pull Request

### Description

* Is this a feature or a fix?
* Feature
* Where should the reviewer start?
* Review the arrow functions in app and how they are passed as props.
* What functionalities do these changes affect? Why were these changes made? What was the motivation behind these changes?
* This allow users to click a poster and change the page to view the dummy details page. Users can click the header logo to return to the main page.  
* What outside features or research did you use to implement this fix?
* How should this be tested?
Check to see that the clicking is working. We should also check if the user agent styles are still acting up after the merge. 

***

#### Please review considerations below:
